build.sbt

```diff
@@ -27,6 +27,7 @@ version := "0.1"
 libraryDependencies += "org.typelevel" %% "cats" % "0.9.0"
 libraryDependencies += "org.scalactic" %% "scalactic" % "3.0.4"
```

Comment:
This is most likely a `Test` dependency as well.

Reply:
will fix!